packages/stardog-graphql-language-server/src/GraphQlLanguageServer.ts

```diff
@@ -181,6 +352,48 @@ export class GraphQlLanguageServer extends AbstractLanguageServer<
       }
     });
 
+    // show relationship completions if the only candidate is a field
+    if (hasFieldCandidate && !hasNamedTypeCandidate) {
```

Comment:
Since you check `hasFieldCandidate && !hasNamedTypeCandidate` together more than once, and since the code will be self-documenting if you do it, I have a slight preference for creating a variable: `const onlyCandidateIsField = hasFieldCandidate && !hasNamedTypeCandidate`. Then, substitute it where necessary.